### PR TITLE
couple of glitchs resolved

### PR DESCRIPTION
**Pull request by memoglr**

### CODE REVIEW

File: shopyo/modules/contact/templates/contact/contact_form.html [hunk 1]
```diff
@@ -11,6 +11,9 @@
 
 {% block content %}
 
+<a href="{{url_for('contact.dashboard')}}"><button class="btn btn-info">view contact dashboard</button></a>
+<br><br>
+
```
Comment (Abdur-rahmaanJ) [inline shopyo/modules/contact/templates/contact/contact_form.html:14]: Contact form should be public facing. Normal users should not see the dashboard button. Maybe add {% if current_user.is_admin ...

File: shopyo/modules/page/templates/page/all_pages.html [hunk 2]
```diff
@@ -16,6 +16,9 @@
 {% endblock %}
 
 {% block content %}
+{%include 'page/nav.html'%}
+<br>
```
Comment (Abdur-rahmaanJ) [inline shopyo/modules/page/templates/page/all_pages.html]: This also add {%if admin, as all_pages is supposed to be viewed from the public

**Review by Abdur-rahmaanJ — CHANGES_REQUESTED** [review]

**Comment by Abdur-rahmaanJ** [comment]
Else good

**Comment by memoglr** [comment]
@Abdur-rahmaanJ please take a look.

File: shopyo/modules/page/info.json [hunk 3]
```diff
@@ -4,5 +4,5 @@
         "type": "show",
         "fa-icon": "fa fa-store",
         "url_prefix": "/page",
-        "panel_redirect": "/dashboard"
+        "panel_redirect": ""
 }
```
Comment (Abdur-rahmaanJ) [inline shopyo/modules/page/info.json:8]: The /dashboard is good. As if it is defined, control panel will redirect to dashboard instead of /contact

**Comment by Abdur-rahmaanJ** [comment]
Thanks for PR!

**Pull request merged**